test/src/components/paginator.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {render, screen} from '@testing-library/preact';
+import userEvent from '@testing-library/user-event';
+import {PaginatorContextProvider} from '~/components/paginator/paginator-context';
+import {PaginatorControls} from '~/components/paginator/search-results/paginator';
+import SimplePaginator from '~/components/paginator/simple-paginator';
+
+function activePage() {
+    const activeButton = screen.getByRole('button', {current: 'page'});
+
+    return activeButton.textContent;
+}
+
+describe('paginator', () => {
```

Comment:
Didn't have a describe wrapper on it before, so it updates all the lines. Only had one test before; now two, so not a big file anyway.